### ISSUE TITLE
Support for AnimatedSprite3D

### ISSUE THREAD

**Issue by mhashirshahzad** [Goutte/godot-addon-animated-shape-2d#17]
Hye, i wanna use this for an HD-2D game

**Comment by Goutte** [comment]
It might work ; since there are no union types (yet), things might be a little weird, but it appears that `AnimatedSprite3D` also has the properties and signals we need.

What kind of collision node would the addon write to, though ?
Do you use `CollisionShape2D` ?

**Comment by mhashirshahzad** [comment]
i am using `CollisionShape3D` present inside `CharacterBody3D`.

**Comment by Goutte** [comment]
Thanks.  How do you envision creating the volume of the `CollisionShape3D` ?

Extrusion of the flat polygon of a `CollisionShape2D` ?

**Comment by mhashirshahzad** [comment]
i dont think so it would be an extrusion of collisionShape2d, it will be just the simple area3d or collisionshape3d

**Comment by MHashirShahzad** [comment]
I am trying to edit it for 3D
![Sprite Frames 2D](https://github.com/user-attachments/assets/b32f3170-e35f-459e-8684-e9579ddf8b02)

Can you tell me why, the create new buttons is not appearing
![Sprite Frames 3D](https://github.com/user-attachments/assets/c5680021-e57c-4176-9d34-20cdb6cd6651)


![image](https://github.com/user-attachments/assets/ab953c22-7769-4d53-9822-68005d7d4d02)

**Comment by mhashirshahzad** [comment]
![image](https://github.com/user-attachments/assets/2ff685ee-1809-4e65-8092-ed873e96229d)
It is overlaying the sprites over one another
